core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -177,7 +178,29 @@ class Translations : LinkedHashMap<String, TranslationEntry>(){
         percentCompleteOfLanguages = TranslationFileReader.readLanguagePercentages()
 
         val translationFilesTime = System.currentTimeMillis() - startTime
-        println("Loading percent complete of languages - "+translationFilesTime+"ms")
+        println("Loading percent complete of languages - ${translationFilesTime}ms")
+    }
+    
+    fun getConditionalOrder(language: String): String {
+        return getText(defaultConditionalOrderingString, language, null)
+    }
+
+    /** Returns the equivalent of a space in the given language
+     * Defaults to a space if no translation is provided
+     */
+    fun getSpaceEquivalent(language: String): String {
+        val translation = getText("\" \"", language, null)
+        return translation.substring(1, translation.length-1)
+    }
+    
+    fun shouldCapitalize(language: String): Boolean {
+        return get(shouldCapitalizeString, language, null)?.get(language)?.toBoolean() ?: true
```

Comment:
This requires protection against translators translating to "true" or "false" in their own language. The toBoolean can crash.
Also it should probably be explicit in the translation file, put like "'true' or 'false' IN ENGLISH"

Reply:
According to the documentation I could find in Android Studio, `.toBoolean()` only checks if it's "true", and if it's anything else it returns `false`, so no crashes should occur. I will add the note in the template.properties though

Reply:
Oh, nice!
I was going by toInt, I know that crashes 😅

Reply:
You can Ctrl-B on that and check the source - yes it's non-throwing. Use toBooleanStrict for the crashing version.